tests/publiccloud/aistack_basic.pm

```diff
@@ -0,0 +1,89 @@
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use publiccloud::utils qw(is_byos registercloudguest register_openstack);
+use publiccloud::ssh_interactive 'select_host_console';
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+use File::Basename;
+
+sub run {
+    my ($self, $args) = @_;
+    #Download unpublished DMS package and install rpm
+    my $ins_rke2 = get_var('RKE2_URL');
+    my $ins_kubectl = get_var('KUBECTL_URL');
+    my $ins_helm = get_var('HELM_URL');
+    #my $add_repo = get_var('HELM_REPO');
+   
+
+    #Create aws instance
+    my $provider;
+    my $instance;
+    select_host_console();
+
+    my $aws_test = get_var('PUBLIC_CLOUD_AWS_TEST', 'basic_sanity');
```

Comment:
Being (Private) AI the effective context needing such test, IMHO related parameters should also evoke and reference to AI, having for publiccloud specific parameter with similar meanings.
This is why I suggest here and elsewhere needed in that project, a different name than `...AWS_TEST`, where even AWS could in future be another CSP, to be more related to the AI project.
It should mean: _p.c.instance available/1 or not/0_, like i.e. longer but clear `PUBLIC_CLOUD_INSTANCE_FOR_AI` or else.

Reply:
Fixed